quarantine.gemspec

```diff
@@ -14,9 +14,10 @@ Gem::Specification.new do |s|
   s.summary = 'Quarantine flaky Ruby Rspec tests'
   s.homepage = 'https://github.com/flexport/quarantine'
   s.license = 'MIT'
-  s.files = Dir['{lib}/**/*', '*.md', '*.gemspec']
+  s.files = Dir['{lib, bin}/**/*', '*.md', '*.gemspec']
+  s.executables = ['quarantine_dynamodb']
 
   s.add_dependency('aws-sdk', '~> 2.11.41')
-  s.add_dependency('rspec', '~> 3.8.0')
+  s.add_dependency('rspec', '>= 3.0', '<= 4.0')
```

Comment:
Do we forsee this breaking in future rspec versions? If not then maybe just have `>= 3.0` for this and `rspec-retry`